rest/admin_api_test.go

```diff
@@ -1217,6 +1243,11 @@ func TestPurgeWithSomeInvalidDocs(t *testing.T) {
 }
 
 func TestReplicateErrorConditions(t *testing.T) {
+
+	if !base.UnitTestUrlIsWalrus() {
```

Comment:
Can we file a ticket to track the disabled tests to re-enable?

Reply:
See https://github.com/couchbase/sync_gateway/issues/2977.  I will add as we come across, but it's also possible to reverse search on `base.UnitTestUrlIsWalrus()` to get current list